### ISSUE TITLE
Possible cycle error

### ISSUE THREAD

**Issue by xeske** [darvid/stashvision#5]
I just did some testing, and noticed that while cycle through sets ("shift + alt + c"), exactly 1 item do not cycle (always highlighted). This stubborn item always has the utmost upper left position.

**Comment by darvid** [comment]
is this with the latest version (0.1.2)?

**Comment by xeske** [comment]
yep

**Comment by xeske** [comment]
https://i.imgur.com/UOQFilx.png
https://i.imgur.com/Xu0ypFn.png
https://i.imgur.com/jDVhvyI.png

**Comment by darvid** [comment]
Ah, I think this was introduced because of the fix for working with ilvl 75+ items (#4)...

So basically the old behavior was to prevent using items over ilvl 74 altogether, breaking the recipe even if you had at least one low item level item, and the rest 75+. The new behavior is to find *one* valid unid rare within the right ilvl range (60-74) and then cycle the *rest* of the items in the stash tab with 60+ (with no upper limit).

One way to fix this would be to find *multiple* "first items" (i.e. items within 60-74), which is doable, but I think the better solution is something I had planned to look into but would require considerable development: that is, either provide a hotkey to re-index items from your stash tab after you remove them and sell sets, or detect when sets are sold (or at least removed from the stash) and re-index automatically. The server currently re-indexes automatically every 10 seconds to avoid hitting rate-limiting with the PoE API.

TLDR: Try waiting at least 10s after selling a set and then re-run the hotkey (remember to hit Esc to clear the overlay after using it once), I think cycling is going to be bricked until I figure out a solution to the issues described above. :( Hopefully that still works, and if that doesn't, then that's another bug entirely. I don't have enough rares in standard atm to thoroughly test cycling, but I will try to find a solution to this issue first if possible.

**Comment by xeske** [comment]
No problem, glad I could help xD

**Comment by darvid** [comment]
can you try this build?

https://github.com/darvid/stashvision/suites/815771597/artifacts/8944296

* extract to the **stashvision-go** directory
* delete the **stashvision-go/stash_items.bleve** directory

**Comment by darvid** [comment]
should be fixed in v0.1.3

https://github.com/darvid/stashvision/releases/tag/v0.1.3

**Comment by xeske** [comment]
Nice! will test at blood aqueduct ;)
________
yep, all correct!
